### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by Have-Samuel**
Mobile Menu Project We: 

1. Applied the Linters
2. Docemented Proffesionally 
3. Applied best practices of HTML &CSS
4. Used the Figma IU
5. Applied JS to make the page dynamic

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -84,7 +84,41 @@ <h4>Multi-Post Stories</h4>
 </div>
```
Comment (Meltrust) [inline index.html:84]: - [ ] On your readme file, please, kindly include the "getting started" section with instructions about how to run the project, as in the Microverse readme template.  That way, your project will be well documented 💪

File: index.html [hunk 2]
```diff
@@ -84,7 +84,41 @@ <h4>Multi-Post Stories</h4>
...
   </section>
 
-  <div class="section-cards">
+  <div>
+    <div class="project section-cards">
```
Comment (Meltrust) [inline index.html:88]: - [ ] On desktop, the "see live" button is misaligned.  Kindly correct it so it looks like the Figma:

Your Implementation | Figma Design
:---:|:---:
image | image

That way, your desktop popup will be consistent 💪

Comment (Meltrust) [inline index.html:88]: On mobile:

- [ ] Please kindly add some margin or padding to the top of the popup content, so it looks like the Figma.
- [ ] Please align the buttons so they match the previous text alignment (see picture).
- [ ] Please make your popup a little bigger in height, so it matches the Figma.

Your Implementation | Figma Design
:---:|:---:
image | image


If you implement these changes, your project will look more like the Figma 👍

File: index.html [hunk 3]
```diff
@@ -84,7 +84,41 @@ <h4>Multi-Post Stories</h4>
...
+        typesetting, remaining essent
+      </p>
+    </div>
+    <div class="button-pop">
+      <button><a href="#"></a> See Live<span><img src="./images/popup1.svg" alt="popupbutton"></span></button>
+      <button><a href="#"></a>See source<span><img src="./images/Github1.png" alt="github"></span></button>
+    </div>
+  </section> -->
+  
+  <!-- <div class="section-cards">
```
Comment (Meltrust) [inline index.html:121]: - [ ] This file contains a huge piece of commented code.  Kindly remove it so the repo becomes cleaner 💪

File: script.js [hunk 4]
```diff
@@ -7,15 +8,180 @@ const menuLinks = document.querySelector('.navmenu');
...
+  title: 'Multi Post Stories',
+  techs: ['html', 'Bootstrap', 'Ruby on rails'],
+  imgSrc: './images/popup.png',
+  description:
+    "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+  livelink: 'https://have-samuel.github.io/mobile-ver',
+  sourceLink: 'https://github.com/Have-Samuel/mobile-ver',
+};
+
+// const popupWindow = document.querySelector('.popup1');
```
Comment (Meltrust) [inline script.js:109]: - [ ] Please, kindly remove this piece of old commented code, so the repo gets cleaner 💪

File: script.js [hunk 5]
```diff
@@ -7,15 +8,180 @@ const menuLinks = document.querySelector('.navmenu');
...
+  }
+  // <button>
+  const projectButton = document.createElement('button');
+  projectButton.classList = 'see-button';
+  projectButton.innerText = 'See project';
+
+  section.appendChild(projectButton);
+  // when clicked shows Id
+  projectButton.addEventListener('click', () => {
+    console.log(project.projectId);
```
Comment (Meltrust) [inline script.js:89]: According to general Javascript best practices requirements:

image

Therefore:

- [ ] Please, kindly do not push to the repo any `console.log` calls.

File: script.js [hunk 6]
```diff
@@ -7,15 +8,180 @@ const menuLinks = document.querySelector('.navmenu');
...
+}
+
+creatPopupCard();
+
+const popupWindow = document.getElementsByClassName('popup1')[0];
+const windowClose = document.querySelectorAll('.see-button');
+
+for (let i = 0; i < windowClose.length; i += 1) {
+  windowClose[i].addEventListener('click', () => {
+    console.log('Helo');
```
Comment (Meltrust) [inline script.js:179]: According to general Javascript best practices requirements:

image

Therefore:

- [ ] Please, kindly do not push to the repo any `console.log` calls.

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi STUDENT_MENTION

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To highlight:

- The popup works ✔️
- Nice responsive implementation 

You can do it

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Hamzaoutdoors — APPROVED** [review]
Hi @Have-Samuel, 

## Congratulations! 🎉 Project Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀

To Highlight:

- [x] The Pull Request has a proper title and description. ✔️
- [x] You have added a descriptive Readme file. Good Job. ✔️
- [x] There is no linter error is present. ✔️
- [x] Project has met all the technical aspects you needed for this project! ✔️ 

_Cheers and Happy coding and Keep rocking!_ 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**